analysis/include/ModuleMapper.h

```diff
@@ -10,167 +10,71 @@
-         * @param key 
-         * @return std::string 
-         */
-        std::string getHwFromSw    (const std::string& key)  {return sw_to_hw[key];};
-
-        /**
-         * @brief Get the Sw From Hw
-         * 
-         * @param key 
-         * @return std::string 
-         */
-        std::string getSwFromHw    (const std::string& key)  {return hw_to_sw[key];};
-
-        /**
-         * @brief Get the String From Hw
-         * 
-         * @param key 
-         * @return std::string 
-         */
-        std::string getStringFromHw(const std::string& key)  {return hw_to_string[key];};
-
-        /**
-         * @brief Get the String From Sw
-         * 
-         * @param key 
-         * @return std::string 
-         */
-        std::string getStringFromSw(const std::string& key)  {return sw_to_string[key];};
-        
-
-        /**
-         * @brief Get the Hybrid Strings
-         * 
-         * @return std::vector<std::string> 
-         */
-        std::vector<std::string> getHybridStrings();
-
-        /**
-         * @brief Get list of string modules.
-         * 
-         * @param strings 
-         */
-        void getStrings (std::vector<std::string>& strings) {
-            for (strmap_it it = string_to_hw.begin(); it!= string_to_hw.end(); ++it)
-            strings.push_back(it->first);
-        }
-        
-        /**
-         * @brief Get list of hw names
-         * 
-         * @param hws 
-         */
-        void getHws (std::vector<std::string>& hws) {
-            for (strmap_it it = hw_to_string.begin(); it!= hw_to_string.end(); ++it)
-            hws.push_back(it->first);
-        }
-
-        /**
-         * @brief Get list of sw names
-         * 
-         * @param sws 
-         */
-        void getSws (std::vector<std::string>& sws) { 
-            for (strmap_it it = hw_to_string.begin(); it!= hw_to_string.end(); ++it)
-            sws.push_back(it->first);
-        }
-
-        /**
-         * @brief Build global svt id map for all FebHybrid combinations
-         * 
-         * @return std::map<std::string, std::map<int,int>> 
-         */
-        std::map<std::string, std::map<int,int>> buildChannelSvtIDMap();
-
-        /**
-         * @brief Return global svt id for channel by providing local channel number and F<n>H<m> of channel
-         * 
-         * @param channel 
-         * @param hwTag 
-         * @param svtid_map 
-         * @return int 
-         */
-        int getSvtIDFromHWChannel(int channel, std::string hwTag, std::map<std::string,std::map<int,int>> svtid_map);  
-
-        /**
-         * @brief Used to generate apv channel map and read in thresholds from database
-         * 
-         */
-        void buildApvChannelMap();
-
-        /**
-         * @brief description
-         * 
-         * @param filename 
-         */
-        void ReadThresholdsFile(std::string filename);
-
-        /**
-         * @brief description
-         * 
-         * @param feb 
```

Comment:
You are going to need to merge with master to make sure you don't lose all Sarah's recent work in adding these comments for Doxygen.